src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -215,22 +296,19 @@ func runClientProxy(cfg *config.Config) {
 	settings.Configure(cfg, version, buildDate)
 	proxiedsites.Configure(cfg.ProxiedSites)
 
-	hqfd := cfg.Client.HighestQOSFrontedDialer()
-	if hqfd == nil {
-		log.Errorf("No fronted dialer available, not enabling geolocation, stats or analytics")
-	} else {
-		// An *http.Client that uses the highest QOS dialer.
-		hqfdClient := hqfd.NewDirectDomainFronter()
-
-		geolookup.Configure(hqfdClient)
-		statserver.Configure(hqfdClient)
-		analytics.Configure(cfg, false, hqfdClient)
-	}
+	updateClientDirectFronter(cfg, mch)
+	wdc := mch.makeWithClient()
+	geolookup.Configure(wdc)
+	statserver.Configure(wdc)
```

Comment:
Local in what sense?  Is the client's own IP used at all if the query includes an IP?  That is, if I do a geolookup query from my home to request the City of a peer in NY do I get a different response than if you do a query from your home?

Where is the server side code for geo.getiantem.org?